src/Command/MarkMarsScientistAsMissingOrDeadCommand.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
```

Comment:
Consistency. Raz robisz przerwę, a raz nie. U nas się nie robi, ale jakbyś miał wszedzie tak samo to bym sie nie przywalił ;p